### PR TITLE
Enhancements for Cross-Chain-Activity chart

### PR DESCRIPTION
**Pull request by valentinoConti**
### Description

This PR implements the mobile design for the cross-chain-activity sankey charts and it also fixes a bug where devices with high resolutions had a pixelated chart.

Big desktop
big desktop

Small desktop
small desktop

Mobile (showing before & after bug fix)
HD

### CODE REVIEW

File: src/components/atoms/Select/index.tsx [hunk 1]
```diff
@@ -44,6 +44,8 @@ const Select = ({
       value={value}
       onChange={value => onValueChange(value)}
       noOptionsMessage={() => noOptionsMessage}
+      menuPortalTarget={document.body}
+      styles={{ menuPortal: base => ({ ...base, zIndex: 1000 }) }}
```
Comment (valentinoConti) [inline src/components/atoms/Select/index.tsx:48]: This two lines are to fix this:

src

File: src/components/molecules/CrossChainChart/styles.scss [hunk 2]
```diff
@@ -82,70 +82,158 @@
...
 
         &.left {
           cursor: pointer;
         }
 
         &.right {
-          border-left: 4px solid var(--color-information-100);
+          .chain-name {
+            opacity: 1;
+            margin-right: -4px;
```
Comment (raop155) [inline src/components/molecules/CrossChainChart/styles.scss:206]: This is not doing anything, we can remove it.

File: src/utils/useWindowSize.tsx
Comment (raop155) [inline src/utils/useWindowSize.tsx:1]: I think putting this file into a `hooks` folder is better. What do you think?

**Review by raop155 — CHANGES_REQUESTED** [review]
Few observations:

Mobile:

image
- The left border should be wider
- Display % on the right side
ref: https://www.figma.com/file/kVuUyVjFFcRP55xHxHoKtH/WormScan?node-id=1648-22830&t=ThtW2JtrzGVJSIpO-0

Desktop:

image

- The minimum height should be larger

ref: https://www.figma.com/file/kVuUyVjFFcRP55xHxHoKtH/WormScan?node-id=1654-25061&t=BOqLHpE83zCZoR0g-0

File: src/components/molecules/CrossChainChart/styles.scss (hunk 2 above)
Comment (valentinoConti) [reply src/components/molecules/CrossChainChart/styles.scss:206]: It is doing something 🙏🏼 

https://user-images.githubusercontent.com/41705567/232526629-9f88e7ba-f6c8-4ca2-98bf-6316b2eab25c.mov

File: src/utils/useWindowSize.tsx
Comment (valentinoConti) [reply src/utils/useWindowSize.tsx:1]: Done ✅

**Comment by valentinoConti** [comment]
@raop155 

👉🏼 The left border should be wider

Done ✅ 

---

👉🏼 Display % on the right side

I talked with Sofi regarding that, we agreed to not show the percentages on the right on mobile for now.

---

👉🏼 The minimum height should be larger

We also agreed that Figmas minimum height is so big, but I moved from 18px to 20px to make it a lil bigger

File: src/components/molecules/CrossChainChart/styles.scss [hunk 3]
```diff
@@ -113,7 +113,7 @@
           background: withOpacity(var(--color-primary-100), 0.2);
           gap: 12px;
           border-left: 0;
-          min-height: 18px;
+          min-height: 20px;
```
Comment (raop155) [inline src/components/molecules/CrossChainChart/styles.scss:116]: The `icons` are 24x24 that's why probably the min-height should be at least 28px or 32px.

**Review by raop155 — APPROVED** [review]
LGTM

**Pull request merged**